tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/TasksApplication.kt

```diff
@@ -3,20 +3,33 @@ package live.ditto.compose.tasks
 import android.app.Application
 import live.ditto.Ditto
 import live.ditto.DittoIdentity
+import live.ditto.DittoLogLevel
+import live.ditto.DittoLogger
 import live.ditto.android.DefaultAndroidDittoDependencies
+import live.ditto.compose.tasks.DittoHandler.Companion.ditto
 
 
 class TasksApplication: Application() {
 
-    companion object {
-        var ditto: Ditto? = null;
-    }
-
     override fun onCreate() {
         super.onCreate()
+        setupDitto()
+    }
+
+    private fun setupDitto() {
         val androidDependencies = DefaultAndroidDittoDependencies(applicationContext)
-        val identity = DittoIdentity.OfflinePlayground(appID = "YOUR_APP_ID", dependencies = androidDependencies);
-        ditto = Ditto(androidDependencies, identity)
+        // Create an instance of DittoSyncKit
```

Comment:
The `DittoSyncKit` name is obsolete; it's just "Ditto" now.

Reply:
removed